src/screens/regimen_setup/currentProducts/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+
+import Button from '../../../components/button';
+import { useThemeContext } from '../../../theme';
+import applyScale from '../../../utils/applyScale';
+import { Container, Image, Content, Title, SubContent } from './style';
+
+interface Iprops {
+  title: string;
+}
+
+interface stateType {
+  display: 'flex' | 'none';
+  added: ``;
+}
+
+export default function CurrentProduct(props: Iprops) {
+  const { colors, fonts } = useThemeContext();
+  const [state, setstate] = useState<'flex' | 'none'>('flex');
```

Comment:
here too

Reply:
Sorry for the duplication, this is the one am using, i declared a type for it earlier